e2e/tests/room/08-roominfo.spec.ts

```diff
@@ -268,5 +268,35 @@ describe('Room info screen', () => {
 					.withTimeout(10000);
 			});
 		});
+
+		describe('Navigate to user status-test', () => {
+			it('Back to rooms list view', async () => {
+				await tapBack();
+				await sleep(300);
+				await tapBack();
+				await sleep(300);
+				await tapBack();
+				await sleep(300);
+			});
+			it('should see the status text with show more label', async () => {
+				const statusTextExpected =
+					'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Etiam laoreet odio lectus, nec varius nisi semper ut porta ante';
+				await navigateToRoomInfo('status-test');
+				await waitFor(element(by.id(`collapsible-text-truncated-${statusTextExpected}`)))
+					.toBeVisible()
+					.withTimeout(10000);
+				await sleep(400);
+				// This is the expected text on CircleCi for both platforms
+				const textWithShowMore =
+					'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Etiam laoreet odio lectus, nec varius... Show more';
+				await waitFor(element(by[textMatcher](`${textWithShowMore}`)))
+					.toExist()
+					.withTimeout(10000);
+				await element(by.id(`collapsible-text-truncated-${statusTextExpected}`)).tap({ x: 320, y: 24 });
```

Comment:
`element(by.label(/[a-z]+/i));`

https://wix.github.io/Detox/docs/api/matchers/#bylabellabel

Reply:
To use the regex, I need to update the detox to 20.11 - https://github.com/wix/Detox/releases/tag/20.11.0